include/dragonbox/dragonbox.h

```diff
@@ -2243,8 +2343,8 @@ namespace jkj::dragonbox {
                 }
             }
 
-            static compute_mul_result compute_mul(carrier_uint u,
-                                                  cache_entry_type const& cache) noexcept {
+            static JKJ_CONSTEXPR20 compute_mul_result
+            compute_mul(carrier_uint u, cache_entry_type const& cache) noexcept {
```

Comment:
Looks like formatting got messed up.

Reply:
But please ignore it if that's what clang-format wants to do with this.

Reply:
Yes, this is clang-format's doing.